pkg/shell/topnav.jsx

```diff
@@ -130,7 +130,7 @@ export class TopNav extends React.Component {
 
         docs.forEach(e => {
             docItems.push(<DropdownItem key={e.label} href={e.url} target="blank" rel="noopener noreferrer" icon={<ExternalLinkAltIcon />}>
-                {e.label}
+                {_(e.label)}
```

Comment:
This smells like it should trigger our static code checks, as it's not a valid xgettext-parseable string. I.e. it would be better to write this as `cockpit.gettext(e.label)`. But as there are no complaints, fine.